frontend/src/components/Collections/components/Grid/components/Row/CollectionRow/index.tsx

```diff
@@ -66,6 +67,11 @@ const CollectionRow: FC<Props> = (props) => {
 
   const isPrivate = visibility === VISIBILITY_TYPE.PRIVATE;
 
+  const handleRowClick = (e: React.MouseEvent) => {
```

Comment:
It seems like a navigation that's not an anchor link is not a11y friendly, and nested anchor links is an antipattern: https://ux.stackexchange.com/questions/102252/more-info-button-vs-making-the-entire-table-row-a-link

Can you help work with @hthomas-czi to prevent the nesting please?

I'm thinking maybe we can just make the name column an anchor element instead of the whole row or something!

Thank you!

Reply:
I'll be removing the DOI anchor to avoid nested anchors and will try changing the entire row to an anchor

Reply:
Thanks Seve! Sorry I forgot to mention that I think the UX stackexchange link seems to suggest that making the entire row an anchor is an antipattern in UX world too 😆 @hthomas-czi PTAL too thank you!

Reply:
HTML semantics actually doesn't allow anything but a td or tr as a child of table, so I'll be putting an anchor in each individual cell

Reply:
Got it! I think we need to resolve the UX level issue first before we resolve the eng implementation. The UX issue here is that when an entire row is clickable, it's unclear if clicking in different column navigates to different pages or the same page, thus the UX antipattern and the Stackexchange discussion!

Reply:
@seve Ah, if we can't make the entire row the anchor element, I agree with @tihuan that the expected behavior is unclear. Can we return to the [original design](https://www.figma.com/file/6wPUe08qSM8V9G2G0UfYOP/Consumer-Collection?node-id=478%3A0) where title and DOI and links?